### PR TITLE
Add -Werror=implicit-fallthrough to cmake build

### PR DESCRIPTION
**Pull request by RipleyTom**

### CODE REVIEW

File: rpcs3/rpcs3qt/rpcn_settings_dialog.cpp [hunk 1]
```diff
@@ -845,6 +846,7 @@ void rpcn_account_edit_dialog::change_password()
 			QMessageBox::information(this, tr("Password Successfully Changed!"), tr("Your password has been successfully changed!"), QMessageBox::Ok);
 		}
 	}
+	[[fallthrough]];
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/rpcn_settings_dialog.cpp:849]: just add a regular break in the scope above

File: rpcs3/rpcs3qt/rpcn_settings_dialog.cpp [hunk 2]
```diff
@@ -801,6 +801,7 @@ void rpcn_account_edit_dialog::change_password()
 			QMessageBox::information(this, tr("Password Reset Token Sent!"), tr("The reset password token has successfully been sent!"), QMessageBox::Ok);
 		}
 	}
+	[[fallthrough]];
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/rpcn_settings_dialog.cpp:804]: why does this fallthrough?

File: rpcs3/rpcs3qt/rpcn_settings_dialog.cpp [hunk 3]
```diff
@@ -844,6 +845,7 @@ void rpcn_account_edit_dialog::change_password()
 
 			QMessageBox::information(this, tr("Password Successfully Changed!"), tr("Your password has been successfully changed!"), QMessageBox::Ok);
 		}
+		[[fallthrough]];
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/rpcn_settings_dialog.cpp:848]:
```suggestion
		break;
```

File: rpcs3/rpcs3qt/rpcn_settings_dialog.cpp [hunk 4]
```diff
@@ -800,6 +800,7 @@ void rpcn_account_edit_dialog::change_password()
 
 			QMessageBox::information(this, tr("Password Reset Token Sent!"), tr("The reset password token has successfully been sent!"), QMessageBox::Ok);
 		}
+		[[fallthrough]];
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/rpcn_settings_dialog.cpp:803]: why does this fallthrough?

Comment (RipleyTom) [reply rpcs3/rpcs3qt/rpcn_settings_dialog.cpp:803]: if no => send token and then ask for token
if yes => ask for token
I could technically move the yes code out of the switch but it's fine as it is.

File: rpcs3/rpcs3qt/emu_settings.cpp [hunk 5]
```diff
@@ -1235,9 +1235,8 @@ QString emu_settings::GetLocalizedSetting(const QString& original, emu_settings_
 		case CELL_SYSUTIL_LANG_ENGLISH_GB: return tr("English (UK)", "System Language");
 		case CELL_SYSUTIL_LANG_PORTUGUESE_BR: return tr("Portuguese (Brazil)", "System Language");
 		case CELL_SYSUTIL_LANG_TURKISH: return tr("Turkish", "System Language");
-		default:
-			break;
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/emu_settings.cpp:1239]: removing this makes no sense

Comment (RipleyTom) [reply rpcs3/rpcs3qt/emu_settings.cpp:1239]: It does nothing, at best it silences warning of missing enum values in the switch which you wouldn't want anyway.

**Pull request merged**